### ISSUE TITLE
Transaction Question

### ISSUE THREAD

**Issue by 592da** [typeorm/typeorm#4539]
**Issue type:**

[x ] question
[ ] bug report
[ ] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ ] `mssql`
[ ] `mysql` / `mariadb`
[ ] `oracle`
[ x] `postgres`
[ ] `cockroachdb`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[x ] `latest`
[ ] `@next`
[ ] `0.x.x` (or put your version here)

**Steps to reproduce or a small repository showing the problem:**

Hello All, 

For this question, let's assume that I have the entities called `Order` and `OrderItems` (Many Order Items to One Order).

What I am trying to achieve is when I am updating the order / inserting new Order Items related to the specific order, queries that trying to read the data, related to those resources, (specific ones and not the whole tables) will be queue, and will be run only after I will complete the transaction.

In other words,
If I am currently updating Order id 1, and simultaneously trying to query Order id 1, to update / read the order or any of its order items, the query that started after will be hold until the first transaction will be finished.

Hope I was clear enough...

Is it something doable with TypeORM ? if so, some code sample will be great :)

Thanks for @peerlock and the amazing community around this great project,
Daniel

**Comment by Kononnable** [comment]
https://github.com/typeorm/typeorm/blob/master/docs/transactions.md#specifying-isolation-levels 
You're looking for transaction isolation levels. It is possible in typeorm to specify which one should be used.

**Comment by imnotjames** [comment]
For other questions, please check out the [community slack](https://join.slack.com/t/typeorm/shared_invite/zt-gej3gc00-hR~L~DqGUJ7qOpGy4SSq3g) or check [TypeORM's documentation page on other support avenues](https://typeorm.io/#/support) - cheers!
